pkg/util/validation/validation.go

```diff
@@ -80,12 +82,13 @@ func IsValidLabelValue(value string) []string {
 		errs = append(errs, MaxLenError(LabelValueMaxLength))
 	}
 	if !labelValueRegexp.MatchString(value) {
-		errs = append(errs, RegexError(labelValueFmt, "MyValue", "my_value", "12345"))
+		errs = append(errs, RegexError(labelValueErrMsg, labelValueFmt, "MyValue", "my_value", "12345"))
 	}
 	return errs
 }
 
 const dns1123LabelFmt string = "[a-z0-9]([-a-z0-9]*[a-z0-9])?"
+const dns1123LabelErrMsg string = "a valid DNS (RFC 1123) label must consist of lower case alphanumeric character or '-', and must start/end with an alphanumeric character"
```

Comment:
Pluralize the first `characters`: `must consist of lower case alphanumeric characters or ...`